frontend/app/src/components/Sidebar/SidebarNav.tsx

```diff
@@ -140,8 +147,26 @@ const SidebarNav = ({
   const [expanded, setExpanded] = useState(false)
   const { pageLinkBaseUrl } = useContext(AppContext)
 
+  useEffect(() => {
+    const cachedSidebarNavExpanded =
+      localStorageAvailable() &&
+      localStorage.getItem("sidebarNavState") === "expanded"
```

Comment:
Nit: The way the `handleViewButtonClick` logic is written, don't think you have to check if `=== "expanded"`
If its expanded, you set the item - if collapsed, you remove it. 
localStorage.getItem returns null if they key does not exist.

Reply:
I have a slight preference for something explicit. I originally wanted `true`, but since local storage uses a string, it didn't make sense. Let me know if you feel strongly.